lab-jinho/test/restaurant-route-test.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const request = require('superagent');
+const expect = require('chai').expect;
+
+require('../server.js');
+
+describe('Restaurant Routes', function() {
+  var restaurant = null;
+
+//POST: test 400 should respond with 'bad request' if no request body was provided or body was invalid
+  describe('POST: /api/restaurant', function(){
+    it('respond with bad request - POST', function(done){
+      request.post('localhost:3000/api/restaurant')
+      .send({invalid:'invalid body'})
+      .end((err, res) => {
+        expect(res.status).to.equal(400);
+        expect(res.text).to.equal('bad request');
+        done();
+      });
+    });
+  });
+//POST: test 200 should respond with the body content for post request with valid body
+  describe('POST: /api/restaurant', function() {
+    it('should return a restaurant', function(done) {
```

Comment:
Minor nitpick- it's a good idea to include the status code you're expecting within your it block. It's just easier to read when you run your tests in the terminal.